runelite-client/src/main/java/net/runelite/client/plugins/timetracking/clocks/ClockPanel.java

```diff
@@ -64,12 +64,13 @@
 
 	private final FlatTextField nameInput;
 	private final IconButton startPauseButton;
-	private final FlatTextField displayInput;
+	protected final FlatTextField displayInput;
```

Comment:
leave this private

Reply:
I made it `protected` because it's used in the subclass `TimerPanel`.